src/main/target/SKYSTARSH743HD/target.c

```diff
@@ -36,15 +36,19 @@ timerHardware_t timerHardware[] = {
     DEF_TIM(TIM4, CH1, PD12, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 2),   // S3
     DEF_TIM(TIM4, CH2, PD13, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 3),   // S4
 
-    DEF_TIM(TIM5, CH1, PA0, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 4),   // S5  
+#if defined(SKYSTARSH743HD_SERVOS)
```

Comment:
I see some targets with both TIM_USE_MC_SERVO and TIM_USE_MC_MOTOR on the same pin.

Wouldn't that be better than a different target for servos?

Reply:
I hadn't noticed that before, but I see that is true.

Looking at the code, it appears that both flags can be used so long as the number of required motors are assigned to lower-numbered PWM outputs.
That is, it wouldn't work to have both MC_MOTOR and MC_SERVO on S1 if you have any motors. But it should work in this case.  Thanks.

I'll update the PR.

Reply:
@mmosca  I see that in many such cases, such as the Matek boards, the pins that are declared MC_MOTOR | MC_SERVO have their own timers. Therefore there cannot be a conflict with other PWM on the same timer.

As discussed on Discord, testing shows that CAN cause / allow a conflict in cases like this, where there is more than one output for that timer.

I am updating the target to use the approach @DzikuVx  suggested.